src/server/resources/queries/vehicle/VEHICLE_NEW_CONTINGENCY_MOT.sql

```diff
@@ -0,0 +1,31 @@
+select veh.registration, date_format(mtc.issued_date,'%Y%m%d') as issued_date, mtc.number
+from vehicle veh, model_detail md,
+  (select max(id) as id, vehicle_id  from mot_test_current
+   group by vehicle_id
+   limit 100000) as latest_mot,
+   mot_test_current mtc,
+   mot_test_emergency_reason as mter
+where veh.model_detail_id = md.id
+and mter.id = mtc.id
+and veh.id = latest_mot.vehicle_id
+and mtc.id = latest_mot.id
+and mtc.status_id not in (4,5) -- exclude vehicles whose latest status is under test or failed
+and date_add(mtc.issued_date, interval 31 day) > curdate()
```

Comment:
I take this to mean that you are only considering contingency tests entered in the last 31 days.

This might not work too well with anonymised data sets since there age and will quickly have no rows satisfying this condition.